ui/package.json

```diff
@@ -2,109 +2,77 @@
-    "mathjax": "^3.0.0",
-    "mathjax-full": "^3.0.0",
+    "less": "^3.10.3",
+    "less-loader": "^7.3.0",
+    "mathjax": "^3.1.2",
+    "mathjax-full": "^3.1.2",
     "pdfjs-dist": "^2.2.228",
-    "query-string": "^6.8.1",
+    "prettier": "^2.2.1",
+    "query-string": "^6.14.1",
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "reselect": "^4.0.0"
-  },
-  "scripts": {
-    "start": "node scripts/start.js",
-    "dev": "PROXY='http://localhost:3000' node scripts/start.js",
-    "build": "node scripts/build.js",
-    "test": "react-scripts test --env=jsdom"
+    "react-hot-loader": "^4.13.0",
+    "reselect": "^4.0.0",
+    "rimraf": "^3.0.2",
+    "style-loader": "^2.0.0",
+    "ts-loader": "^8.0.17",
+    "typescript": "^3.5.3",
+    "webpack": "^4.29.5",
+    "webpack-cli": "^4.5.0",
+    "webpack-dev-server": "^3.11.2"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "babel": {
-    "presets": [
-      "react-app"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "jest": {
-    "collectCoverageFrom": [
-      "src/**/*.{js,jsx,ts,tsx}",
-      "!src/**/*.d.ts"
-    ],
-    "setupFiles": [
-      "react-app-polyfill/jsdom"
-    ],
-    "setupFilesAfterEnv": [],
-    "testMatch": [
-      "<rootDir>/src/**/__tests__/**/*.{js,jsx,ts,tsx}",
-      "<rootDir>/src/**/*.{spec,test}.{js,jsx,ts,tsx}"
-    ],
-    "testEnvironment": "jest-environment-jsdom-fourteen",
-    "transform": {
-      "^.+\\.(js|jsx|ts|tsx)$": "<rootDir>/node_modules/babel-jest",
-      "^.+\\.css$": "<rootDir>/config/jest/cssTransform.js",
-      "^(?!.*\\.(js|jsx|ts|tsx|css|json)$)": "<rootDir>/config/jest/fileTransform.js"
-    },
-    "transformIgnorePatterns": [
-      "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|ts|tsx)$",
-      "^.+\\.module\\.(css|sass|scss)$"
-    ],
-    "modulePaths": [],
-    "moduleNameMapper": {
-      "^react-native$": "react-native-web",
-      "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy"
-    },
-    "moduleFileExtensions": [
-      "web.js",
-      "js",
-      "web.ts",
-      "ts",
-      "web.tsx",
-      "tsx",
-      "json",
-      "web.jsx",
-      "jsx",
-      "node"
-    ],
-    "watchPlugins": [
-      "jest-watch-typeahead/filename",
-      "jest-watch-typeahead/testname"
-    ]
-  },
-  "devDependencies": {
-    "@types/classnames": "^2.2.9",
-    "@types/cookie": "^0.4.0",
-    "@types/dompurify": "^2.0.0",
-    "@types/node": "^12.12.9",
-    "@types/pdfjs-dist": "^2.1.2",
-    "@types/qs": "^6.9.5",
-    "@types/react": "^16.8.23",
-    "@types/react-dom": "^16.8.5",
-    "less": "^3.10.3",
-    "less-loader": "^5.0.0",
-    "node-sass": "^4.13.1",
-    "react-scripts": "^3.3.0",
-    "rewire": "^4.0.1",
-    "typescript": "^3.5.3"
-  }
+  "devDependencies": {}
```

Comment:
Technically, we don't need to separate the dependencies between `dependencies` and `devDependencies` since there's no intent to distribute this as an NPM package. The Skiff template puts everything in `dependencies` so I kept with that, but I have no strong feelings toward either.

Reply:
I also do not have a strong opinion either way. Keeping everything in `dependencies` is fine with me.

Reply:
I think the other reason I know of to use dev dependencies is to keep them out of the production image if there's ever a server-side component to things. That said, probably not going to happen anytime soon so I'm okay with collapsing them if you want.